ModularTegustation/tegu_traitor_code/tegu_traitor_items.dm

```diff
@@ -69,3 +69,102 @@
 	new /obj/item/radio/headset/syndicate/alt(src)
 	new /obj/item/clothing/gloves/combat(src) //1 TC?
 	new /obj/item/gun/ballistic/automatic/pistol(src) //7 TC
+
+// Advanced hypno-flash. For psychologists.
+/obj/item/assembly/flash/hypnotic/adv
+	desc = "A modified flash device, used to beam preprogrammed orders directly into the target's mind."
+	light_color = COLOR_RED_LIGHT
+	var/hypno_cooldown = 29 SECONDS // Big damn cooldown
+	var/hypno_cooldown_current
+	var/hypno_message // User can change it by alt-clicking the flash.
```

Comment:
The default hypno flash has you say a message afterwards, you should mention in its description and in the message when buying in the uplink, that this is a hypnosis you have to write the message to, beforehand

Reply:
Changed in uplink text.